doc/posts/2023/2023_06_05_vara_week2.rst

```diff
@@ -0,0 +1,33 @@
+Deep Dive into VQ-VAE : Week 2
+=====================
```

Comment:
====  too short, it should be the size as the title above, otherwise it generates warning